OOP/ActionableMenuItem.java

```diff
@@ -0,0 +1,25 @@
+import java.io.IOException;
+
+
+
+public class ActionableMenuItem implements MenuItem{
```

Comment:
ACTION: As u know u have many menu-items like 'passenger', 'yes', 'no' etc. Shouldn't there be a separate MenuItem class for each of them as each of them might have different actions.